.classpath

```diff
@@ -28,5 +28,10 @@
 	<classpathentry exported="true" kind="lib" path="lib/jsoup-1.18.1.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/vinnie-2.0.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/ez-vcard-0.12.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="/hibiscus/lib/pdfbox/pdfbox-3.0.2.jar"/>
```

Comment:
Ich hoffe, dass wir nicht mal eine neuere Version benötigen, als Hibiscus liefert bzw. falls sie in Hibiscus aktualisiert wird, unser Code noch funktioniert.

Reply:
Ja, das stimmt schon, aber es gibt ja auch schon andere Bibliotheken  die wir von Hibscus verwenden (zB. Itext) da ist es ja genauso. Solange es kein Problem gibt würde ich es so lassen, sonst müssen wir die richtige Version halt in JVerein extra hinzufügen.